packages/utils/src/source-map.ts

```diff
@@ -87,23 +90,17 @@ export function parseSingleFFOrSafariStack(raw: string): ParsedStack | null {
     return null
   }
 
-  // eslint-disable-next-line regexp/no-super-linear-backtracking, regexp/optimal-quantifier-concatenation
-  const functionNameRegex = /((.*".+"[^@]*)?[^@]*)(@)/
-  const matches = line.match(functionNameRegex)
-  const functionName = matches && matches[1] ? matches[1] : undefined
-  const [url, lineNumber, columnNumber] = extractLocation(
-    line.replace(functionNameRegex, ''),
-  )
-
-  if (!url || !lineNumber || !columnNumber) {
+  // eslint-disable-next-line regexp/no-super-linear-backtracking,regexp/no-misleading-capturing-group,regexp/no-unused-capturing-group
+  const matches = /(at\s+)?(async\s+)?([^\s@]+)?(?:@|\s+\()?([^\s()]+):(\d+):(\d+)\)?/.exec(line)
```

Comment:
Disabling these ESLint rules is not convincing. Can we fix these errors as well?

Reply:
can you check @AriPerkkio  ? check also discord server, some hints there